### ISSUE TITLE
Optimizer Less is considering Full URL as a relative URL.

### ISSUE THREAD

**Issue by pedramphp** [raptorjs/optimizer-less#4]
```
background: url(' http://p.ebaystatic.com/aw/promo/deals/spr_DailyDeals.png ' );
```

is throwing the following an error

Caused by: Error: Error while resolving resource URL for path "/Users/mpedramrazi/rpp/webapp/ui-comps/ebayui/refit/itemcard/lib/http:/p.ebaystatic.com/aw/promo/deals/spr_DailyDeals.png". Error: Error: File with path "/Users/mpedramrazi/rpp/webapp/ui-comps/ebayui/refit/itemcard/lib/http:/p.ebaystatic.com/aw/promo/deals/spr_DailyDeals.png" does not exist

**Comment by patrick-steele-idem** [comment]
New version published with fix: `optimizer-less@2.0.2`

**Comment by pedramphp** [comment]
Thank you
